Compiling for Windows

I use VS Code in windows10 & installed PlatformIO,
Do I have to make certain settings before compiling the code? 
for Example add some Define?
or add another lib?

**Errors:**
```
.pio\libdeps\pico32_LIGHT_EDITION\TFT_eSPI\Processors/TFT_eSPI_ESP32.h:79:36: fatal error: SPIFFS.h: No such file or directory
.
.
.
src\apps\_experiments\runtime_test.cpp:10:23: error: 'MINI_IOT_DEVICENAME' was not declared in this scope
src\apps\_experiments\runtime_test.cpp:10:44: error: 'MINI_IOT_SERVER' was not declared in this scope
src\apps\_experiments\runtime_test.cpp:10:61: error: 'WIFI_SSID' was not declared in this scope
src\apps\_experiments\runtime_test.cpp:10:72: error: 'WIFI_PASS' was not declared in this scope
```

--- a/Readme.md
+++ b/Readme.md
@@ -9,3 +9,5 @@ Then clone this repository
 and open it with VSC
 
     code open-smartwatch-os
+
+rename file `include/config.h.example` to `include/config.h` and adapt the values according to your requirements.

